.github/actions/are-format-checks-disabled/action.yml

```diff
@@ -0,0 +1,67 @@
+# **********************************************************
```

Comment:
> The original PR [#7345](https://github.com/DynamoRIO/dynamorio/pull/7345) was reverted because github.event.pull_request._links.commits.href is absent in case of push/merge.
> his PR handles the push/merge case by using
> if ${{ contains(toJSON(github.event.commits.*.message), 'DISABLE_FORMAT_CHECKS') }}

Suggest: Use backtics to make the code separately formatted